docs/content/guide/ie.ngdoc

```diff
@@ -13,8 +13,8 @@ addressing issues specific to IE8 or earlier.
 </div>
 
 This document describes the Internet Explorer (IE) idiosyncrasies when dealing with custom HTML
-attributes and tags. Read this document if you are planning on deploying your Angular application
-on IE.
+attributes and tags. Read this document if you plan on users accessing your Angular application
```

Comment:
I am not a big fun of this either. How about _"...if you plan on supporting your Angular application on IE."_ ?